release-notes/release-notes502.md

```diff
@@ -7,7 +7,7 @@ keywords: releasenotes newfeatures deleted technologypreview bugfixes knownissue
 
 ## Overview
 
-The Release Notes provide high-level coverage of the improvements and additions that have been implemented in TinyMCE 5.0.2 and document known problems in this release, as well as notable bug fixes, deprecated functionality, and other details.
+{{site.productname}} 5.0.2 was released for {{site.enterpriseversion}} and {{site.cloudname}} on Friday, March 22<sup>nd</sup>, 2019. It includes {{site.productname}} 5.0.2 and additional changes to premium plugins. The Release Notes provide high-level coverage of the improvements and additions that have been implemented in TinyMCE 5.0.2 and document known problems in this release, as well as notable bug fixes, deprecated functionality, and other details.
```

Comment:
This also seems to be incorrect. Looking at the slack announcements it was Monday 18th March. The change done on March 22 was a premium plugin hotfix.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1779/commits/bb176495886196c47aa96ca142dc929a0bd67fae